react-native-libraries.json

```
@@ -11787,5 +11787,27 @@
     "ios": true,
     "android": true,
     "newArchitecture": true
+  },
+  {
+    "githubUrl": "https://github.com/gunnartorfis/react-native-week-month-date-picker",
```

```suggestion
    "githubUrl": "https://github.com/gunnartorfis/react-native-reanimated-toasts",
```

Looks like you have copied over the other repository URL.